.github/workflows/create-pr.yaml

```diff
@@ -19,6 +20,51 @@ permissions:
   contents: read
 
 jobs:
+  create-tentants-config-pr:
+    if: >
+      inputs.run_for == 'tentants-config' || inputs.run_for == 'both' || inputs.run_for == ''
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout tenants-config
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        repository: redhat-appstudio/tenants-config
+        ref: main
+        path: tenants-config
+
+    - name: Update ec-policies
+      run: ./hack/update-infra-deployments.sh ../tenants-config
```

Comment:
Oh now I see this is needed for https://github.com/enterprise-contract/ec-policies/pull/810